docs/source/bricks.rst

```diff
@@ -380,8 +383,7 @@ Determines if the section of text exceeds the specified caps ratio. Used in
 ``is_possible_narrative_text`` and ``is_possible_title``, but can be used independently
 as well. You can set the caps threshold using the ``threshold`` kwarg. The threshold
 defaults to ``0.3``. Only runs on sections of text that are a single sentence.
-You can also set the threshold using the ``NARRATIVE_TEXT_CAP_THRESHOLD`` environment
-variable. The environment variable takes precedence over the kwarg. The caps ratio
+The environment variable takes precedence over the kwarg. The caps ratio
```

Comment:
Don't we want to give the name of the env variable here (it now seems to be `UNSTRUCTURED_NARRATIVE_TEXT_CAP_THRESHOLD`)?

Reply:
Ooops actually meant to get rid of that section since the env var applies to `is_possible_narrative_text` and not `exceeds_caps_ratio`. Fixed